common/src/main/java/com/microsoft/identity/common/internal/dto/AccessTokenRecord.java

```diff
@@ -262,12 +262,14 @@ public void setExpiresOn(final String expiresOn) {
         mExpiresOn = expiresOn;
     }
 
-    private boolean isExpired(final String expires) {
+    private boolean isExpired(final String expiresEpochSeconds) {
+        //TODO: Explain why this is not just
+        // return System.currentTimeMillis() < Long.valueOf(expiresEpochSeconds) * 1000;
```

Comment:
Let's take your change here; IIRC this code was ganked out of ADAL

Reply:
And we can drop the private method, too,